### ISSUE TITLE
[Yarn.social]: Add new default pod logo

### ISSUE THREAD

**Issue by prologic** [jointwt/twtxt#385]

**Comment by sorenpeter** [comment]
Yarn.social Logo ideas
================

Stitching the letters on a piece of cardboard and some holes (dots):
![image](https://user-images.githubusercontent.com/8458421/107821747-46237300-6d7d-11eb-88a9-9c51546fd9d4.png)

```

.   .
 \ /  Y A R N
  |   .social
  .

---------------

.   .
 \ /  
  |  YARN
  .social

-------------
.  .
 \/  
 /   YARN
.  .social


------------------

.   . _   __  _  .        
 \ / /_\  |_) |\ |   
  | /   \ | \ | \|
  .       .   .social 

--------------------
_   _  _   __  _  _
 \ /  /_\  |_) |\ |
  |  /   \ | \ | \|

      .social

--------------------

\   /    __  \  |
 \ / /\  |_) |\ |
  / /__\ | \ | \|
 / /    \|  \|  \
             .social 

--------------------

.     .
 \   /    __  .  .
  \ / /\  |_) |\ |
   / /__\ | \ | \|
  / /    \|  \|  \
 . .      .   .social 

--------------------

.   .    __  .  .
 \ / /\  | \ |\ |
  / /__\ |_/ | \|
 / /    \| \ |  \
  .      .   .social 

--------------------
    _____
   /_____\
  /Y A R N\
 (_________)
  \.social/
   \_____/

---------------
    _____
   /     \
  /Y A R N\
  \.social/
   \_____/

```

Slogans
========

(in random rotation under the logo)

"Stitching together your own timeline"

"Weaving your own network"

"Weaving a feed of your own"

"Spinning your own server"

"No strings attached"

"Only follow threads you want"

# And some more hand-drawn logo ideas

![IMG_20210212_212947](https://user-images.githubusercontent.com/8458421/107821827-681cf580-6d7d-11eb-92e5-ec8c2933050a.jpg)

![IMG_20210212_215359](https://user-images.githubusercontent.com/8458421/107821836-6f440380-6d7d-11eb-9466-c5dd41f19e33.jpg)

**Comment by prologic** [comment]
What about the idea of incorporating a ball of yarn that we had a little while ago? With purple as part of the color scheme ?￼

**Comment by prologic** [comment]
Yeah the concept you have 15 rows (_counting the cells of your paper_) is good. "Yarn" in a circle with lines that look like parts of a ball fo yarn. I like that. Encapsulates both "ball of yarn" and "yarning circle"

**Comment by sorenpeter** [comment]
When i count to the rows to 15 i'm not sure i end up the same place as you. I should have given them numbers or something.
Will you make a comment with a cropped image?

**Comment by sorenpeter** [comment]
I didn't have my purple pen handy...

**Comment by prologic** [comment]
This one:

<img width="111" alt="Screen Shot 2021-02-13 at 07 32 51" src="https://user-images.githubusercontent.com/1290234/107824735-b1953100-6dcd-11eb-882a-f974efa6ad1b.png">

I like this direction so far 👌

**Comment by sorenpeter** [comment]
🧶 .social
https://emojipedia.org/yarn/
and talk at: https://twtxt.net/conv/tgqyilq
![](https://i.pinimg.com/originals/08/6f/52/086f52641ec9bc88c09c00aa39edfc70.png)

**Comment by sorenpeter** [comment]
Not my work, but just for inspiration: https://dribbble.com/shots/1675477-Sphere-Rings
